### ISSUE TITLE
Documentation of the FLINT Cloud

### ISSUE THREAD

**Issue by AayushSaini101** [moja-global/FLINT.Cloud#121]
The documentation of the flint cloud is missing. The curl file only contains a little information.
**Missing content of the documentation.**
![new](https://user-images.githubusercontent.com/60972989/170866939-c89a1ea5-8c68-4832-b40d-5141f688e275.PNG)
- Project Structure 
- Internal working.
- Required output of the API.

**Comment by ayush78781** [comment]
@padmajabhol as you were already working on Flint-Cloud. So, please do share your thoughts how we can make east going.

**Comment by soma2000-lang** [comment]
@AayushSaini101 can I work on this

**Comment by padmajabhol** [comment]
> @padmajabhol as you were already working on Flint-Cloud. So, please do share your thoughts how we can make easy going.

Hi, I must have missed your notif. can you take a look at the curl commands here? https://github.com/moja-global/FLINT.Cloud/blob/master/local/rest_api_gcbm/curl.md
I'm pretty sure this will make more sense?

**Comment by ayush78781** [comment]
> > @padmajabhol as you were already working on Flint-Cloud. So, please do share your thoughts how we can make easy going.
> 
> Hi, I must have missed your notif. can you take a look at the curl commands here? https://github.com/moja-global/FLINT.Cloud/blob/master/local/rest_api_gcbm/curl.md I'm pretty sure this will make more sense?

Yes it does make sense. But it's not being updated on the current documentation of Flint Cloud.

**Comment by ayush78781** [comment]
> @AayushSaini101 can I work on this

Hii @soma2000-lang You can work on this issue. But I would recommend you to explore Flint.cloud project before starting with it. And list out your understanding here of its various api's and their endpoints also the tech blockers you face in the process. Then we can go ahead assigning different parts of it as per the interest. I hope it makes sense. Feel free to ask your queries either here or over the cloud channel in slack.

**Comment by ayush78781** [comment]
@HarshCasper can we use SwaggerHUb for an Interactive documentation of rest_apis like [this](http://petstore.swagger.io/). SwaggerHub Editor uses YAML, but we can import and download both YAML and JSON. By implementing it one doesn't have to set-up the whole project on their local or gcp. They can easily interact and understand various request through Swagger-UI. Do let me know your thoughts over it. If every thing goes good I can move ahead implementing it.

**Comment by ayush78781** [comment]
> @HarshCasper can we use SwaggerHUb for an Interactive documentation of rest_apis like [this](http://petstore.swagger.io/). SwaggerHub Editor uses YAML, but we can import and download both YAML and JSON. By implementing it one doesn't have to set-up the whole project on their local or gcp. They can easily interact and understand various request through Swagger-UI. Do let me know your thoughts over it. If every thing goes good I can move ahead implementing it.

@AayushSaini101 What's your thoughts over this.

**Comment by chicken-biryani** [comment]
@ayush78781 will you be okay with posting the question over slack?
